kcef/src/main/kotlin/dev/datlag/kcef/step/fetch/PackageDownloader.kt

```diff
@@ -50,7 +50,11 @@ internal data object PackageDownloader {
 
         val success = download.client.prepareGet(downloadUrl) {
             onDownload { bytesSentTotal, contentLength ->
-                progress.downloading((bytesSentTotal.toFloat() / contentLength.toFloat()) * 100F)
+                if (contentLength == null) {
+                    progress.downloading(0f)
+                } else {
+                    progress.downloading((bytesSentTotal.toFloat() / (contentLength.toFloat()) * 100F))
+                }
```

Comment:
Happy to tweak this handling. In practice I've not seen `contentLength` be null in my testing so I'm hoping this isn't a big deal